src/config.ts

```diff
@@ -0,0 +1,62 @@
+import { makeConfig } from 'cleaner-config'
+import { asArray, asNumber, asObject, asOptional, asString } from 'cleaners'
+
+export const asConfig = asObject({
+  couchUri: asOptional(asString, 'http://admin:password@localhost:5984'),
+  httpPort: asOptional(asNumber, 8008),
+  infoServerAddress: asOptional(asString, 'info1.edge.app'),
+  infoServerApiKey: asOptional(asString, 'hex'),
+  bridgeCurrencies: asOptional(asArray(asString), ['USD', 'BTC']),
+  cryptoCurrencyCodes: asOptional(asArray(asString), [
+    'BTC',
+    'ETH',
+    'USDT',
+    'XRP',
+    'DOT',
+    'ADA',
+    'LINK',
+    'LTC',
+    'BNB',
+    'BCH',
+    'XLM',
+    'DOGE',
+    'USDC',
+    'UNI',
+    'AAVE',
+    'WBTC',
+    'BSV',
+    'EOS',
+    'XMR',
+    'XEM',
+    'TRX',
+    'SNX',
+    'XTZ',
+    'COMP',
+    'THETA',
+    'MKR',
+    'SUSHI',
+    'ATOM',
+    'UMA',
+    'VET'
+  ]),
+  fiatCurrencyCodes: asOptional(asArray(asString), [
+    'EUR',
+    'CNY',
+    'JPY',
+    'GBP'
+  ]),
+  ratesServerAddress: asOptional(asString, 'https://rates1.edge.app'),
+  currencyConverterApiKey: asOptional(asString, ''),
+  coinMarketCapHistoricalApiKey: asOptional(
```

Comment:
Again, let's make the defaults blank strings, to reduce the amount of hassle for editors of the file, and then put the `XXXX-XXXX` placeholders as comments (if we even need them). Defaults that are actually useful (like the currency lists) can stay as they are.